apps/i18n/common/en_us.json

```diff
@@ -733,6 +733,8 @@
   "notAnswered": "Not Answered",
   "notStarted": "Not started",
   "nPoints": "{numPoints, plural, one {1 point} other {# points}}",
+  "numMultipleChoiceCorrect": "# multiple choice correct",
```

Comment:
nit: the strings here look consistent (capitalize the 1st letter of the string, lowercase for the rest of the words), but the student name column isn't consistent with this (it's "Student Name"). i'm not sure if that's a string you have control over (maybe it's pre-existing and used elsewhere), but the capitalization should be consistent across the columns

Reply:
In the spec it is "Student Name", but you are correct in pointing this out because the other columns in the spec follow the convention you mention.  I will follow up with Poorva.